### PR TITLE
[ID] Update `People/Performance_Points_Committee`

### PR DESCRIPTION
**Pull request by mousesalt**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Performance_Points_Committee/id.md [hunk 1]
```diff
@@ -10,16 +7,30 @@ tags:
...
 
-Panel ini pertama kali diperkenalkan kepada publik melalui [artikel berita *Performance Points & Star Rating Updates* pada bulan November 2021](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) sebagai bentuk tanggapan dari hasil [survei komunitas](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) yang diselenggarakan beberapa bulan sebelumnya.
+Panel dari osu! pertama kali diperkenalkan kepada publik melalui [artikel berita *Performance Points & Star Rating Updates* pada bulan November 2021](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) sebagai bentuk tanggapan dari hasil [survei komunitas](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) yang diselenggarakan beberapa bulan sebelumnya.
```
Comment (Nivalyx) [inline wiki/People/Performance_Points_Committee/id.md:10]:
```suggestion
Panel mode osu! pertama kali diperkenalkan kepada publik melalui [artikel berita *Performance Points & Star Rating Updates* pada bulan November 2021](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) sebagai bentuk tanggapan dari hasil [survei komunitas](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) yang diselenggarakan beberapa bulan sebelumnya.
```

File: wiki/People/Performance_Points_Committee/id.md [hunk 2]
```diff
@@ -10,16 +7,30 @@ tags:
...
+
+Panel dari osu!mania dibentuk pada 22 April 2022 dalam mengantisipasi lonjakan upaya aktif perkembangan, dimulai dengan [perubahan dalam perhitungan kesulitan untuk pelepasan hold note di osu!mania](https://github.com/ppy/osu/pull/17913).
```
Comment (Nivalyx) [inline wiki/People/Performance_Points_Committee/id.md:12]:
```suggestion
Panel mode osu!mania dibentuk pada 22 April 2022 untuk mengantisipasi lonjakan pengembangan di ruang lingkup osu!mania, yang dimulai dengan [perubahan algoritma perhitungan tingkat kesulitan pelepasan *hold note*](https://github.com/ppy/osu/pull/17913).
```

Comment (Nivalyx) [reply wiki/People/Performance_Points_Committee/id.md:12]: feel like this way would be much better

File: wiki/People/Performance_Points_Committee/id.md [hunk 3]
```diff
@@ -1,7 +1,4 @@
 ---
-outdated: true
-outdated_since: bbf3f7094b0f7d6a587bffdbdb8beb44a0ecdb69
-tags:
   - pp committee
```
Comment (Joehuu) [inline wiki/People/Performance_Points_Committee/id.md:2]: This was accidentally deleted.
```suggestion
tags:
  - pp committee
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**